spell-check.R

```diff
@@ -7,28 +7,33 @@
 arguments <- commandArgs(trailingOnly = TRUE)
 file_pattern <- "\\.(Rmd|md|rmd)$"
 
+# dictionary is required first argument
+dict_file <- arguments[1]
+
+arguments <- arguments[-1]
```

Comment:
I kind of wonder if we want to just do `optparse` here. It'll add another dependency but maybe it's worth it to be more explicit, especially if we also want to allow for another argument with a  different pattern besides Rmd/rmd/md

Reply:
We could, but I think for the first version this probably makes the most sense.